### ISSUE TITLE
BrowserActivity.tempListenerInstance has leaked

### ISSUE THREAD

**Issue by kitek** [klarna/kco-mobile-sdk#15]
KCO-Android version:  1.6.1
It seems you're not unregister some references properly: 

```
GC ROOT static com.klarna.checkout.browser.BrowserActivity.tempListenerInstance
references com.klarna.checkout.internal.f.b
references com.klarna.checkout.internal.js.interfaces.JSBridgeEvent.controller
references com.klarna.checkout.internal.a.j
Device: Google google Android SDK built for x86 sdk_google_phone_x86
Android Version: 7.0 API: 24 LeakCanary: 1.5 00f37f5
Durations: watch=5015ms, gc=120ms, heap dump=1080ms, analysis=21936ms
Details:
Class com.klarna.checkout.browser.BrowserActivity
static tempListenerInstance = com.klarna.checkout.internal.f@316046192 (0x12d67b70)
static h = null
static $classOverhead = byte[2076]@321556481 (0x132a9001)
Instance of com.klarna.checkout.internal.f
static $classOverhead = byte[324]@321362577 (0x13279a91)
a = java.lang.String@314724992 (0x12c25280)
b = com.klarna.checkout.internal.js.interfaces.JSBridgeEvent@318708536 (0x12ff1b38)
shadow$_klass_ = com.klarna.checkout.internal.f
shadow$_monitor_ = 0
Instance of com.klarna.checkout.internal.js.interfaces.JSBridgeEvent
static $classOverhead = byte[316]@321327617 (0x13271201)
controller = com.klarna.checkout.internal.a@317470336 (0x12ec3680)
eventHandlers = java.util.ArrayList@318708512 (0x12ff1b20)
isMessageFromOverlay = false
unitTestMode = false
shadow$_klass_ = com.klarna.checkout.internal.js.interfaces.JSBridgeEvent
shadow$_monitor_ = -1920337930
Instance of com.klarna.checkout.internal.a
static a = null
static $classOverhead = byte[340]@321216033 (0x13255e21)
```

**Comment by gbanfalvikl** [comment]
Thanks for the heads up, we've solved the issue and it'll be addressed in the next release.

**Comment by omarbh** [comment]
Hi @kitek We have released a fix for this in the version 1.6.5 please try this out and if the issue is resolved then we are going to mark the ticket as resolved.

**Comment by kitek** [comment]
👍
